packages/server/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@aragonone/ipfs-background-service-server",
+  "version": "0.0.0",
+  "license": "GPL-3.0-or-later",
+  "scripts": {
+    "lint": "eslint src/** --ext .ts",
+    "test": "jest",
+    "build": "yarn lint && yarn check-types && yarn build:js",
+    "build:js": "babel ./src --out-dir ./build --extensions .ts --root-mode upward",
+    "build:shared": "lerna run build --scope */*-shared --stream",
+    "build:clean": "yarn clean && yarn build",
+    "clean": "rm -rf ./build",
+    "check-types": "tsc",
+    "start": "node ./build/index.js",
+    "start:dev": "nodemon --exec 'babel-node --extensions .ts --root-mode upward' ./src/index.ts"
```

Comment:
Not sure with yarn workspaces, but it seems a bit weird to not declare the dev dependencies needed for the package in its own package.json (e.g. here, we don't declare `babel`, `eslint`, `nodemon`, `tsc`, etc).

Reply:
It's the same dependency for every package so I didn't want to repeat myself. Also if we need to update typescript/babel  we have to make sure we do it in every package, I'm sure I will mess it up at some point. We always do development on the whole monorepo anyways so I'm not sure about the value of declaring this multiple times

Reply:
I usually feel more comfortable declaring all the dependencies with the package itself, but definitely agree there's a lot of leeway here since all the components of the monorepo are meant to be used together!

If we can, let's leave a note in the README about this—that all shared dev dependencies are declared at the top level.